doc/md/cloud/getting-started.md

```diff
@@ -0,0 +1,220 @@
+          dev-url: postgres://postgres:pass@localhost:5430/test?sslmode=disable
+          ariga-token: ${{ secrets.ARIGA_TOKEN }}
+```
+
+</TabItem>
+<TabItem value="maria">
+
+```yaml {7,11,36,39}
+name: Atlas CI
+on:
+  # Run whenever code is changed in the master branch,
+  # change this to your root branch.
+  push:
+    branches:
+      - master
+  # Run on PRs where something changed under the `path/to/migration/dir/` directory.
+  pull_request:
+    paths:
+      - 'path/to/migration/dir/*'
+jobs:
+  lint:
+    services:
+      # Spin up a maria:10.7 container to be used as the dev-database for analysis.
+      maria107:
+        image: mariadb:10.7
+        env:
+          MYSQL_DATABASE: test
+          MYSQL_ROOT_PASSWORD: pass
+        ports:
+          - 4306:3306
+        options: >-
+          --health-cmd "mysqladmin ping -ppass"
+          --health-interval 10s
+          --health-start-period 10s
+          --health-timeout 5s
+          --health-retries 10
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.1
+        with:
+          fetch-depth: 0 # Mandatory unless "latest" is set below.
+      - uses: ariga/atlas-action@latest
+        with:
+          dir: path/to/migrations
+          dir-format: atlas # Or: golang-migrate, goose, dbmate
+          dev-url: maria://root:pass@localhost:4306/test
+          ariga-token: ${{ secrets.ARIGA_TOKEN }}
+```
+</TabItem>
+
+</Tabs>
+
+6. After merging the workflow to your mainline branch, the workflow will be triggered.
+7. Refresh Ariga Cloud and your project will appear!
+
+### Viewing CI Runs
+In the system, you can view all the CI runs that were triggered by the Atlas GitHub workflow.
+A run can complete in one of three ways:
+
+1. <img src="https://release.ariga.io/images/assets/success.svg" width="15" /> Successful - the CI ran successfully and found no errors or issues in your SQL statements.
+2. <img src="https://release.ariga.io/images/assets/warning.svg" width="15" /> Issues Found - there are some issues with your SQL statements that may cause a failure in production.
+3. <img src="https://release.ariga.io/images/assets/error.svg" width="15" /> Failed - the CI run failed, and can be caused for a number of reasons (bad SQL statements, incorrect configuration,
+etc.). The root of the cause will be stated in the report itself.
+
+For each of these runs includes the following:
+- A summary of the run
+- SQL statements that were analyzed
+- An ERD that shows the changes made to the schema, as well as a full view
+
+### Inviting Members
+Under 'Settings' > 'Members', you can invite team members to your organization.
+These members will receive an email with a link to Ariga Cloud, and will be required to sign up with the same email
+in order to access the organization.
+
+### Regenerating Tokens
+It is possible to regenerate the access token, however once you do so the old token will be **deprecated**.
```

Comment:
Deprecated means "will be removed soon", but in Nebula it immediatly is invalid, isn't it?

Reply:
correct! nice catch

Reply:
invalidated is the correct term.